content_handler/framebuffer_skia.cc

```diff
@@ -89,7 +90,8 @@ void FramebufferSkia::Bind(mojo::InterfaceHandle<mojo::Framebuffer> framebuffer,
 }
 
 void FramebufferSkia::ConvertToCorrectPixelFormatIfNeeded() {
-  if (info_->format == mojo::FramebufferFormat::ARGB_8888) {
+  if (info_->format == mojo::FramebufferFormat::ARGB_8888 ||
+      info_->format == mojo::FramebufferFormat::RGB_x888) {
```

Comment:
if you're going to RGB_x888, can't you do this more efficiently by just shifting by 8?